### DISCUSSION TITLE
Atlas Scientific Consumer/Lab grad probes in Intellichem 1/2" NPT

### DISCUSSION THREAD

**Discussion by davthomaspilot** [tagyoureit/nodejs-poolController#743] [Q&A]
I still have my Accutrol probes in the Pentair Intellichem flow jar, but now hooked up to REM.  

The price is so low on the Atlas Scientific consumer grade probes, I figured I'd give them a try.  At that price, I'd just swap them out on something like a yearly basis.

I bought the mounting kits that have the pvc T fitting and adapter, but I don't think they will work in the Intellichem flow jar.  The 1/2" male NPT threads fit, but the holes in Intellichem flow jar are recessed.  So, there's no way to tighten the nut around the Atlas probe to squeeze the rubber gland for a waterproof fit.

Or am I missing something?

If there' not a way to make it work, I'll just connect the T fittings with pvc for the probes.

**Comment by rstrouse** [comment]
You can't tighten them down then screw them in?

**Comment by davthomaspilot** [comment]
I tried again--

It was really hard to get the plastic "nipple" out of the flow jar.  Maybe 1/8" of threads are accessible above the lid of the flow jar.

But, I got it out.  This time I tightened the gland nut before I screwed the nipple into the Intellichem.  Water escapes between the probe and the rubber gland.

I made sure I assembled the gland like the video on their website shows.

So, now this has nothing to do with mounting in an Intellichem, since the water is escaping between the gland and the probe.

I think you indicated you used these for a while--did you have a similar problem (I saw you said they went out of calibration quickly).

**Reply by tagyoureit** [reply]
Can you post a picture of the probe/gland?  

And the consumer probes are less accurate and need calibration more often.  When they start to go crazy and have bad readings/can't be calibrated it's a good sign they need to be replaced.

**Reply by davthomaspilot** [reply]
Finally got it--

Really tightened it down this time.  Maybe too far--it seemed to jump a thread on the last bit if tightening.

I figured I'd messed up the threads, but I kept tightening until just before it would slip a thread again.   Now It's not leaking.  Hooray!

Yeah, from what I've recently read, the solution inside the probe will eventually get contaminated through the tip.  The tip is sort of like a sponge or felt tip pen--a little bit of the water being sampled gets in the probe.

As the internal solution of the probe gets contaminated, you can mitigate by calibrating.  A bigger probe with more solution volume (like the  AS industrial grade probe) will last longer and need calibration less often since the percentage of contamination in the bigger volume is smaller.

At least that's the way I understand it.

I don't think the absolute accuracy is that important, unless its grossly off.  But, "going crazy" is  a problem.

Like I said, I figured I'd give them a try.  I may set up so I have both sets of probes, the Accutrol/Intellichem AND the consumer grade AS.  Be interesting to see how they compare, especially over time.

**Reply by rstrouse** [reply]
Did you try the IntelliChem probes on the EZO circuit.  I suspect these are probably very similar in their internals to the AS lab grade probes.

**Reply by davthomaspilot** [reply]
Yes, the Intellichem (same as Accutrol) worked fine with the EZ-ORP.  

The ph probe read ridiculously high (10.x).  But, I suspect I messed up the solder job for the SMA connector on the coax.  I know you said crimp, but I didn't want to buy the crimp tool, since I could just get a pre-made cable from Amazon (BMC to SMA).

That cable will get here today and I'll try it out.  

RIght now, I'm running with Acutrol ORP and Atlas Scientific pH with EZ-ORP and EZ-ph in the Intellichem flow jar.  

I'll compare the ph readings from the AS versus Accutrol versus reagent tests. 

I'm using to absolute ORP readings being different when I get a new probe,  But, I just empirically determine what ORP reading is required to keep any algae from getting started and adjust the set point accordingly.

**Comment by davthomaspilot** [comment]
Using the new BNC to SMA cable brought the pH readings using the Intellichem probe back to something reasonable.

I think the Intellichem probe reads higher--something like 7.8 versus 7.6.  But, I haven't checked either against a reagent recently.   On the todo list (it's raining today).

**Reply by rstrouse** [reply]
If you calibrate to the EZO-ORP/PH that all goes away.  You will get a 3 point calibration curve regardless of the probe.
